BasicTests/blockgenesistest.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
```

Comment:
I have no idea how many teams are still using this files. perhaps it is better to leave it here as it is small